perception/object_recognition/prediction/map_based_prediction/include/cubic_spline.hpp

```diff
@@ -1,37 +1,35 @@
-/*
- * MIT License
- *
- * Copyright (c) 2018 Forrest
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in all
- * copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
- * SOFTWARE.
-*/
-#ifndef CUBIC_SPLINE_H
-#define CUBIC_SPLINE_H
+// Copyright 2018 Forrest
```

Comment:
That's a good question, but probably not for me to answer, since I know basically nothing about licenses. I just changed the format like the linter wanted.

Reply:
Let discuss this in the meeting today, as I this is not the only instance of different licensing or multi licensing (if that is even allowed)

Reply:
@k0suke-murakami 
As I rememeber, I think this code was brought from somewhere else, and that's why we have different license for this specific file. Could you confirm?

Reply:
As per the discussion in the meeting yesterday, we have decided to make these changes on a later date on all effected files and packages.